src/server/api/routes/channel-members.router.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const channelMembersController = require('../controllers/channel-members.controller');
+
+/**
+ * @swagger
+ * /channel-members/{ID}:
+ *  delete:
+ *    summary: Delete a channel member
+ *    description:
+ *      Will delete a channel member with a given ID.
+ *    produces: application/json
+ *    parameters:
+ *      - in: path
+ *        name: ID
+ *        description: ID of the channel member to delete.
+ *    responses:
+ *      200:
+ *        description: channel member deleted
+ *      5XX:
+ *        description: Unexpected error.
+ */
```

Comment:
Would be nice to add the `404` type response here as well 👍  (when the id provided doesn't exist)